lib/scarpe/wv/radio.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
+
+class Scarpe
+  class WebviewRadio < Scarpe::WebviewWidget
+    attr_reader :text
+
+    def initialize(properties)
+      super(properties)
+      @checked = properties[:checked]
+      puts "checked: #{@checked}"
+
+      bind("click") do
+        send_display_event(event_name: "click", target: shoes_linkable_id)
+      end
+    end
+
+    def properties_changed(changes)
+      items = changes.delete("checked")
+      if items
+        html_element.unmark_radio_button
```

Comment:
This looks like it will only ever unmark, never mark. Is that right?

Reply:
Ah, okay. That makes sense. Thanks!

Reply:
Hm. But if we change checked from the Shoes side, it won't get checked. So this will only happen on click.

Reply:
oo hm ok

Reply:
added tht in a way like button so we can mark it or unmark it by clicking it button calls the checked method
![Screenshot_20230710_213526](https://github.com/scarpe-team/scarpe/assets/85360080/7db84a35-7f00-4a80-921c-5b0df5ccd606)

Reply:
Nice!